flexget/plugins/list/trakt_list.py

```diff
@@ -193,10 +193,10 @@ def items(self):
                     continue
                 entry = Entry()
                 if list_type == 'episode':
-                    entry['url'] = 'http://trakt.tv/shows/%s/seasons/%s/episodes/%s' % (
+                    entry['url'] = 'https://trakt.tv/shows/%s/seasons/%s/episodes/%s' % (
                         item['show']['ids']['slug'], item['episode']['season'], item['episode']['number'])
                 else:
-                    entry['url'] = 'http://trakt.tv/%s/%s' % (list_type, item[list_type]['ids'].get('slug'))
+                    entry['url'] = 'https://trakt.tv/%ss/%s' % (list_type, item[list_type]['ids'].get('slug'))
```

Comment:
A couple of lines up `list_type` is defined of which the trailing `s` is stripped, yielding 'serie'/'movie'/'episode'. That variable is use as the key to the Trakt API response dictionary. And it is _also_ used here to construct the URL. Which requires a trailing 's' again.